docs/src/config/File.md

```diff
@@ -72,7 +72,6 @@ volume_controller = "alsa"  # use softvol for macOS
 on_song_change_hook = "command_to_run_on_playback_events"
 
 # The name that gets displayed under the connect tab on
-# official clients. Spaces are not allowed!
```

Comment:
Seems like you dropped a bit much there :sweat_smile: (the first two words are still relevant).